common/src/main/resources/docker/ecs.bom

```diff
@@ -0,0 +1,231 @@
+brooklyn.catalog:
+  version: "2.1.0-SNAPSHOT" # CLOCKER_VERSION
+  publish:
+    description: |
+      Resources for working with Docker and Amazon ECS from Apache Brooklyn
+    license_code: APACHE-2.0
+    overview: README.md
+
+  items:
+
+  - id: ecs-cluster-template
+    name: "EC2 Container Service Docker Cluster"
+    description: |
+      Creates a cluster of Docker engines for use with Amazon ECS
+    itemType: template
+    iconUrl: https://upload.wikimedia.org/wikipedia/commons/thumb/a/af/AWS_Simple_Icons_Compute_ECS.svg/240px-AWS_Simple_Icons_Compute_ECS.svg.png
+    item:
+      services:
+        - type: ecs-cluster
+
+  - id:  ecs-cluster
+    name: "EC2 Container Service Docker Cluster"
+    description: |
+      Creates a cluster of Docker engines for use with Amazon ECS
+    itemType: entity
+    iconUrl: https://upload.wikimedia.org/wikipedia/commons/thumb/a/af/AWS_Simple_Icons_Compute_ECS.svg/240px-AWS_Simple_Icons_Compute_ECS.svg.png
+    item:
+      type: org.apache.brooklyn.entity.stock.BasicApplication
+
+      brooklyn.parameters:
+        # Duplicated parameters for UI visibility
+        - name: docker.initial.size
+          label: "Initial Cluster Size"
+          description: |
+            Size of the docker cluster when created initially
+          type: integer
+          default: 1
+        - name: docker.max.size
+          label: "Maximum Cluster Size"
+          description: |
+            Maximum size the docker cluster can be scaled to
+          type: integer
+          default: 5
+        - name: docker.sharedsecuritygroup.create
+          label: "Create Docker SharedSecurityGroup"
+          description: |
+            Clocker blueprint will configure security groups to allow access between docker nodes and
+            to allow external access to deployed apps
+          type: boolean
+          default: true
+        - name: ecs.cluster.name
+          label: "ECS Cluster Name"
+          description: |
+            The name of the ECS cluster
+          type: string
+          default: "clocker"
+
+      brooklyn.children:
+        - type: docker-cluster
+          id: docker-cluster
+          name: "docker-cluster"
+
+  - id:  docker-cluster
```

Comment:
`docker-cluster` seems a strange name for this, to add to the catalog. We'll have lost the context that it is for EC2 container service. We should probably have "ecs" in the catalog item id.